frontend/src/components/Tree/Tree.jsx

```diff
@@ -46,34 +45,37 @@ function animateTree(nodeGroupEnter, enteringAndUpdatingLinks) {
 function renderTree(dimensions, jsonData, svgRef, onNodeClick) {
   const svg = select(svgRef.current);
   const { width, height } = dimensions;
-  const marginLeft = 70;
-  const marginTop = 30;
+
+  const radius = width < height ? width : height;
 
   // Transform hierarchical data
-  const root = hierarchy(jsonData);
-  const treeLayout = tree().size([height, width]);
+  const root = hierarchy(jsonData).sort((a, b) =>
+    ascending(a.data.name, b.data.name)
+  );
+  const treeLayout = tree()
+    .size([2 * Math.PI, radius / 3])
+    .separation((a, b) => (a.parent === b.parent ? 1 : 2) / a.depth);
 
   // Creates the links between nodes
-  const linkGenerator = linkHorizontal()
-    .x((link) => link.y + marginLeft)
-    .y((link) => link.x + marginTop);
+  const linkGenerator = linkRadial()
+    .source((link) => link.source)
+    .target((link) => link.target)
+    .angle((d) => d.x)
+    .radius((d) => d.y);
 
   // Enrich hierarchical data with coordinates
   treeLayout(root);
 
   // Create the node group, which will hold the nodes and labels
   const nodeGroup = svg.selectAll(".node-group").data(root.descendants());
-  // Append a `g` element, to group SVG shapes together. 
+  // Append a `g` element, to group SVG shapes together.
   // More info at https://stackoverflow.com/questions/17057809/d3-js-what-is-g-in-appendg-d3-js-code
-  const nodeGroupEnter = nodeGroup.enter().append("g"); 
+  const nodeGroupEnter = nodeGroup.enter().append("g");
+  const nodeGroupEnterAndUpdate = nodeGroupEnter.merge(nodeGroup);
 
-  nodeGroupEnter
-    .merge(nodeGroup)
+  nodeGroupEnterAndUpdate
+    //.merge(nodeGroup)
```

Comment:
Can you remove this line if it's not needed anymore?

Reply:
Done